src/renderer/component/fileTile/view.jsx

```diff
@@ -97,16 +98,19 @@ class FileTile extends React.PureComponent<Props> {
           {isResolvingUri && <div className="card__title--small">{__('Loading...')}</div>}
           {!isResolvingUri && (
             <React.Fragment>
+              <div className="card__subtitle-price">
+                <FilePrice uri={uri} />
+              </div>
               <div className="card__title--small card__title--file">
                 <TruncatedText lines={2}>{title || name}</TruncatedText>
               </div>
+              <div className="card__subtext">
+                <TruncatedText lines={3}>{description}</TruncatedText>
+              </div>
               <div className="card__subtitle">
                 {showUri ? uri : channel || __('Anonymous')}
                 {isRewardContent && <Icon icon={icons.FEATURED} />}
                 {showLocal && isDownloaded && <Icon icon={icons.LOCAL} />}
```

Comment:
I think the file price should stay on the bottom. The title should be the first thing they see.

Reply:
IMO it should be:
```
{title}
{channel}

{description}

{price}
```

Reply:
I think that makes sense. This was a special request from Tom. I don't think he was especially attached to it. So no biggie.